modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java

```diff
@@ -16,6 +16,7 @@
 
 import com.liferay.frontend.taglib.clay.servlet.taglib.soy.VerticalCard;
 import com.liferay.frontend.taglib.clay.servlet.taglib.util.LabelItem;
+import com.liferay.petra.string.StringPool;
```

Comment:
A bit of leftover code?

Reply:
Thanks let me check that, I think it's a left over from the rebase.